src/lib/network/network_autoyast.rb

```diff
@@ -334,20 +334,20 @@ def assign_udevs_to_devs(udev_rules)
         key.downcase!
 
         # find item which matches to the given rule definition
-        item, matching_item = LanItems.Items.find do |_, i|
+        item_id, matching_item = LanItems.Items.find do |_, i|
           i["hwinfo"] &&
             (i["hwinfo"]["busid"].downcase == key || i["hwinfo"]["mac"].downcase == key)
         end
         next if !matching_item
 
-        name_from = item_name(item)
+        name_from = LanItems.current_name_for(item_id)
         log.info("Matching device found - renaming <#{name_from}> -> <#{name_to}>")
 
         # rename item in collision
-        rename_lan_item(colliding_item(name_to), name_from)
+        rename_lan_item(LanItems.colliding_name_item_id(name_to), name_from)
```

Comment:
NP: in my POV renaming it to ```colliding_name_item_id``` is a bit overkill

Reply:
Ok, I will ommit rewording it, but just will move

Reply:
I completely agree but we have to survive it as long as ```LanItems::Items``` is here with us.